L1Trigger/L1TTrackMatch/python/L1TrackJetProducer_cfi.py

```diff
@@ -0,0 +1,80 @@
+import FWCore.ParameterSet.Config as cms
+from L1Trigger.VertexFinder.VertexProducer_cff import VertexProducer
+
+#prompt jet selection
+L1TrackJets = cms.EDProducer('L1TrackJetProducer',
+        L1TrackInputTag= cms.InputTag("L1GTTInputProducer","Level1TTTracksConverted"),
+#        L1TrackInputTag= cms.InputTag("L1TrackSelectionProducer", "Level1TTTracksSelected"),
+        VertexInputTag = cms.InputTag("VertexProducerEmu","l1verticesEmulation"),
+        MaxDzTrackPV = cms.double( 1.0 ), #max distance from PV;negative=no cut
+	trk_zMax = cms.double (15.) ,    # maximum track z
+	trk_ptMax = cms.double(200.),    # maximumum track pT before saturation [GeV]
+	trk_ptMin = cms.double(2.0),     # minimum track pt [GeV]
+   	trk_etaMax = cms.double(2.4),    # maximum track eta
+	nStubs4PromptChi2=cms.double(10.0), #Prompt track quality flags for loose/tight
+        nStubs4PromptBend=cms.double(2.2),
+        nStubs5PromptChi2=cms.double(10.0),
+        nStubs5PromptBend=cms.double(2.2),
+	trk_nPSStubMin=cms.int32(-1),    # minimum PS stubs, -1 means no cut
+	minTrkJetpT=cms.double(-1.),     # min track jet pt to be considered for most energetic zbin finding 
+	etaBins=cms.int32(24),
+	phiBins=cms.int32(27),
+	zBins=cms.int32(1),
+	d0_cutNStubs4=cms.double(-1),
+	d0_cutNStubs5=cms.double(-1),
+	lowpTJetMinTrackMultiplicity=cms.int32(2),#used only for more than 1 z-bins (ie not *prompt*)
+        lowpTJetThreshold=cms.double(50.),#used only for more than 1 z-bins (ie not *prompt*)
+	highpTJetMinTrackMultiplicity=cms.int32(3),#used only for more than 1 z-bins (ie not *prompt*)
+        highpTJetThreshold=cms.double(100.),#used only for more than 1 z-bins (ie not *prompt*)
+	displaced=cms.bool(False), #Flag for displaced tracks
+	nStubs4DisplacedChi2=cms.double(5.0), #Displaced track quality flags for loose/tight
+	nStubs4DisplacedBend=cms.double(1.7),
+	nStubs5DisplacedChi2=cms.double(2.75),
+	nStubs5DisplacedBend=cms.double(3.5),
+        nDisplacedTracks=cms.int32(2)
+)
+
+#displaced jets
+L1TrackJetsExtended = cms.EDProducer('L1TrackJetProducer',
+	L1TrackInputTag= cms.InputTag("TTTracksFromExtendedTrackletEmulation", "Level1TTTracks"),
+        VertexInputTag = cms.InputTag("VertexProducerEmu","l1verticesEmulation"),
```

Comment:
l1tVertexProducerEmu

Reply:
I think this is l1tVertexFinderEmulator right?